akka-remote/src/main/resources/reference.conf

```diff
@@ -91,6 +91,16 @@ akka {
       # InetAddress.getLocalHost.getHostName is used if 
       # "<getHostName>" is specified. 
       hostname = "<getHostAddress>"
+
+      # Any message sent remotely to one of these actor paths
+      # is passed over a dedicated stream so that it does not interfere
+      # with "normal" and system messages. Paths does implicitly start
+      # with "/user" so an entry for the actor at /user/supervisor/actor
```

Comment:
I think that implicit "/user" prefix was a mistake for the deploy section.

Also, we might want to use this for some of our own "/system" actors.